### DISCUSSION TITLE
Yield difference between SWIM3 and SoilWater module

### DISCUSSION THREAD

**Discussion by KeLiu7** [APSIMInitiative/ApsimX#8904] [Q&A]
I am trying to use SWIM3 for some simulations, and I found that after replacing the SoilWater module with SWIM3, the barley yield is greatly reduced, only about 5% of using SoilWater module.

To make the two simulations consistent, only when I add `SWIMSetLowerBC` under SWIM3 module, and set the lower boundary condition to `Potential` and the value to the maximum soil depth (1800mm), then the crop growth and yield under the two simulations will be completely same.
![image](https://github.com/APSIMInitiative/ApsimX/assets/72141690/a174dc7c-7e07-4df9-b074-7e2aea64cb9d)

 
But there is a big difference in the Watertable depth between the two simulations. I am not sure whether my approach is correct. If so, could you please explain the reason? If not, could you please give me some hints to revise the settings? Many thanks
![image](https://github.com/APSIMInitiative/ApsimX/assets/72141690/e0baa847-880f-4f58-aa6d-2ce9a26f8a35)

**Comment by hol353** [comment]
I'm not a SWIM expert but I do know that SWIM works best with thin layers, not SoilWater thick layers. Users normally put in a layer structure model that lets you specify lots of thin layers, without having to change the layer structure in Physical and elsewhere. Look at a SWIM example. Setting the lower boundary condition for no reason is probably not the correct approach. I'll leave it to a SWIM expert to comment further.

**Reply by KeLiu7** [reply]
Thanks Dean. I just found the `outputLayers` under `Soils` module. It looks like the `outputLayers` are very similar to the replacement function?

**Reply by hol353** [reply]
You don't want OutputLayers. You want 'LayerStructure' - look in the SWIM example. Yes, it works similarly to replacments in that it replaces the layer structure in Physical, Organic etc with the one you specify in 'LayerStructure' .

**Reply by KeLiu7** [reply]
Many thanks Dean😊

**Comment by hut104** [comment]
Ke,
I am not sure what sort of system you are applying, but I think the graph shows two different bottom boundary conditions (BBC).
The SoilWater graph has a WT at 1800mm.  ie no water table.  It is probably set to free drainage or to have a WT at that depth.  

The SWIM simulation seems to have a water table set to be close to the surface.  If you set the BBC to a potential of 1800mm (ie the water table is 1800mm above that point, the water table will be near to the surface.

**Reply by KeLiu7** [reply]
Thanks Neil. I changed the watertable depth in SWIM3 to 1800mm (i.e. same as SoilWat), but the barley yield is only about 5% of using SoilWater module. I have attached my sims for your reference. 

I'm just curious about what needs to be done to make the simulation results consistent between the SoilWat and SWIM3.

[SWIM3Test.zip](https://github.com/APSIMInitiative/ApsimX/files/15287899/SWIM3Test.zip)
